src/Node/Express/App.purs

```diff
@@ -22,119 +22,118 @@ import Node.Express.Internal.App
 import Node.Express.Handler
 
 
+-- | Monad responsible for application related operations (initial setup mostly).
+data AppM e a = AppM (Application -> Eff e a)
+type App e = AppM (express :: EXPRESS | e) Unit
 
---| Monad responsible for application related operations (initial setup mostly).
-data AppM a = AppM (Application -> ExpressM a)
-type App = AppM Unit
-
-instance functorAppM :: Functor AppM where
+instance functorAppM :: Functor (AppM e) where
     map f (AppM h) = AppM \app -> liftM1 f $ h app
 
-instance applyAppM :: Apply AppM where
+instance applyAppM :: Apply (AppM e) where
     apply (AppM f) (AppM h) = AppM \app -> do
         res <- h app
         trans <- f app
         return $ trans res
 
-instance applicativeAppM :: Applicative AppM where
+instance applicativeAppM :: Applicative (AppM e) where
     pure x = AppM \_ -> return x
 
-instance bindAppM :: Bind AppM where
+instance bindAppM :: Bind (AppM e) where
     bind (AppM h) f = AppM \app -> do
         res <- h app
         case f res of
              AppM g -> g app
 
-instance monadAppM :: Monad AppM
+instance monadAppM :: Monad (AppM e)
 
-instance monadEffAppM :: MonadEff eff AppM where
+instance monadEffAppM :: MonadEff eff (AppM e) where
     liftEff act = AppM \_ -> unsafeInterleaveEff act
```

Comment:
Good catch. This was actually fixed in the other PR, I wonder why it wasn't fixed here.